README.md

```diff
@@ -117,6 +118,7 @@ const advancePlayer = (move, player) =>
     Right: () => ({x: player.x + 1, y: player.y}),
     Down: () => ({x: player.x, y: player.y + 1}),
     Left: () => ({x: player.x - 1, y: player.y}),
+    _: () => (player),
```

Comment:
Changed my mind. Noticed that elsewhere in the docs the parens are not used, so if I really wanted to be consistent on this I should have changed it there too.